database/migrations/2025_02_28_050226_modules_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('modules', function (Blueprint $table) {
```

Comment:
@FredPeal lets change this to kanvasDomain? or Domains? , @arfenis thoughts? if not we stick to this